src/app/content/highlights/components/DisplayNote.tsx

```diff
@@ -88,7 +88,7 @@ const DisplayNote = React.forwardRef<HTMLElement, DisplayNoteProps>((
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [element, confirmationRef, confirmingDelete, textToggle, width, isTocOpen, searchQuery]);
 
-  return <div className={className} ref={mergeRefs(ref, element)} data-highlight-card>
+  return <div className={className} ref={mergeRefs(ref, element)} tabIndex={0} data-highlight-card>
```

Comment:
hmmm i'm not sure about allowing users to tab through the cards with the keybard... i'm not sure if that is desirable or if we'd rather force them to tab through the highlights in the content. we might need bey or victoria to decide on this

https://openstax.slack.com/archives/C8V1XET1R/p1617824288341300